artslam_localizer:
    ros__parameters:
        # topic where the IMU data is read from
        imu_topic: "/imu/data"
        # topic where the pointclouds are read from
        pointcloud_topic: "/ouster/points"
        # topic where the estimate of the pose is published onto
        odom_topic: "/state_estimation"
        # frame where the robot is localized 
        global_frame: "map"
        # frame of the robot
        base_frame: "base_link"
        # frame in which the localizer performs the localization
        sensor_frame: "os_sensor"
        # whether to use IMU
        use_imu: False
        # whether to use Encoders
        use_encoders: False
        # whether to use GPS
        use_gnss: True
        # whether to use Point Clouds (at least one is required)
        use_point_clouds: True
        # use sim time
        use_sim_time: true
